packages/perseus-editor/src/widgets/interactive-graph-editor/interactive-graph-editor.tsx

```diff
@@ -405,6 +405,86 @@ class InteractiveGraphEditor extends React.Component<Props> {
                         />
                     </LabeledRow>
                 )}
+                {this.props.correct?.type === "angle" && (
```

Comment:
This adds the missing options to the Angle Graph Editor for controlling "showAngles" and "allowReflexAngles"